catroid/build.gradle

```diff
@@ -140,6 +140,11 @@ android {
         versionName "0.9.66"
         println "VersionCode is $versionCode"
         println "VersionName is $versionName"
+        buildConfigField "String", "GIT_COMMIT_INFO", "\"${getGitCommitInfo()}\""
+        buildConfigField "String", "MAIN_URL", (project.hasProperty('web_test_url') ?
+                "\"" + project.getProperties().get('web_test_url') + "\"" :
+                '"https://share.catrob.at"')
```

Comment:
The "https://share.catrob.at" is a magic String here -> would be better to have it in own constant with name as before "MAIN_URL_PRODUCTION"